rubrik_cdm/api.py

```diff
@@ -71,20 +71,20 @@ def _common_api(self, call_type, api_version, api_endpoint, config=None, job_sta
         if call_type != 'JOB_STATUS':
             self._api_validation(api_version, api_endpoint)
 
-            # if '/cluster/me/bootstrap' or '/cluster/me/version' in api_endpoint:
-            #     if self.ipv6_addr != "":
-            #         header = {
-            #             'Content-Type': 'application/json',
-            #             'Accept': 'application/json',
-            #             'Host': '[' + self.ipv6_addr + ']'
-            #         }
-            #         self.log('Created boostrap header: ' + str(header))
-            #     else:
-            #         header = {
-            #             'Content-Type': 'application/json',
-            #             'Accept': 'application/json',
-            #         }
-            #         self.log('Created boostrap header: ' + str(header))
+            if '/cluster/me/bootstrap' or '/cluster/me/version' in api_endpoint:
+                if self.ipv6_addr != "":
+                    header = {
+                        'Content-Type': 'application/json',
+                        'Accept': 'application/json',
+                        'Host': '[' + self.ipv6_addr + ']'
+                    }
+                    self.log('Created header: ' + str(header))
+                else:
+                    header = {
+                        'Content-Type': 'application/json',
+                        'Accept': 'application/json',
+                    }
+                    self.log('Created header: ' + str(header))
```

Comment:
Do we need this section? Seems like it would potentially overwrite the standard authentication header and not include the User-Agent string.

Reply:
Well, that's a good question. Neither bootstrap nor version check requires an auth header, but the user agent string would be good to have in there for the version check. I recall adding it for a reason, but it may not be a good one :joy: Let me poke a little bit more to make sure.

Reply:
Ah, I think this is my bad. Testing now. I forgot this code was relocated when it was commented out. It should be removed. Testing now..

Reply:
Removed unnecessary code and tested again.